content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+Each region features its own tick loop (ticking at Minecraft's regular 20TPS), running on a thread pool in parallel.\
+This allows for better performance on servers with many players and entities, who are spread far out.
+
+Folia was created because Minecraft is single-threaded, meaning it cannot take advantage of all of your CPU, and instead can only use a small portion of it.\
+Softwares such as Paper already help to alleviate this issue, but Folia takes it a step further by splitting the world up into regions.
+
+<Callout type="warn" title="Warning">
+    **Folia is not completely usable!**\
+    You must consider whether you need to use it. Folia is not completely stable yet, although it is stable enough to be used in production, one famous example being 2b2t.\
+    You must be prepared to deal with bugs and issues and a high hardware requirement of a (recommended) minimum of 16 CPU cores.
+    Consider using other (more stable) softwares such as [Pufferfish](#pufferfish-recommended) or [Purpur](#purpur) beforehand.
+
+    **Ensure you thoroughly test every feature, command and mechanic in your server while using this software.**
+
+    Additionally, **Folia does not fix lag**, and requires a very specific playerbase where everyone is spread out relatively far.
+    If your players are not spread out then there will be more entities in each region, which will cause more lag, making the experience identical if not worse than using a software such as Paper.
+
+</Callout>
+
+<Card href="https://jar.setup.md/download/folia/latest/latest" title="Download">
+  Download a copy of the latest Folia server software via
+  [jar.setup.md](https://jar.setup.md/).
+</Card>
+
+## MultiPaper
+
+[MultiPaper](https://github.com/MultiPaper/MultiPaper) is a 1.20.1 fork of [Purpur](#purpur) that splits your world(s) into multiple servers, running independently.\
+Each world is split into regions (similar to Folia), and each region runs on its own server.\
+This server software is often known as "sharding", where each server is a "shard".\
+Each server communicates with each other and is able to transfer players between servers, allowing for a seamless experience.
+
+<Callout type="warn" title="Warning">
+    **MultiPaper is highly unstable!**\
+    You must consider whether you need to use it. MultiPaper is notorious for being unstable and is not recommended for production use.\
+    Although many public servers such as DonutSMP use a sharding solution, they are often custom-made and are not available to the public.
+
+    Consider using / testing other (more stable) softwares such as [Pufferfish](#pufferfish-recommended) or [Purpur](#purpur) beforehand.
+    **Ensure you thoroughly test every feature, command and mechanic in your server if you decide to use this software.**
+</Callout>
+
+<Card href="https://multipaper.io/download.html" title="Download">
+  Download a jarfile for MultiPaper here.
+</Card>
+
+## 1.8 Softwares
```

Comment:
"Softwares" sounds really weird to me. Can we call it "Legacy Server Software" instead?

Reply:
Fixed in [7d34f87](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/7d34f87fbe9d4d314cbdaa72a1532d2e51464f93) and [d20c33d](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/d20c33d060a328b2cce5bbeb24c70aee63e38311)